pkgs/development/tools/golangci-lint/default.nix

```diff
@@ -1,6 +1,6 @@
-{ buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+{ buildGo123Module, fetchFromGitHub, lib, installShellFiles }:
```

Comment:
I also think we should always use latest available go to build packages, so maybe default `buildGoModule` to latest version, and for packages that don't like it can use something like `buildGoModule_stable` or the currently available `buildGo122Module`?

Reply:
That could delay adding new go compilers significantly because now go release maintainers have to upgrade the ecosystem in one go.

Reply:
Ah okay, I don't really understand how things work in nixpkgs tbh so you might have the better answer than me. For now the most important thing for me is `golangci-lint` and `gopls` because it has a really nasty habit of freezing my system with all RAM and CPU maxed out whenever I open my LSP enabled Neovim on a project with latest go.mod file so I think these two should always use latest available Go version when possible.

Reply:
govulncheck too 🙏